### ISSUE TITLE
Research - Comparison of multiple Large Language Base Models

### ISSUE THREAD

**Issue by dominic0df** [amosproj/amos2024ss08-cloud-native-llm#15]
## User story
1. As a software architect
2. I want / need to make a comparison of different Large Language Models and identify a collection that fits best to the goal of creating a LLM for Kubernetes
3. So that our created LLM will perform best.

## Acceptance criteria
* Different Large Language Base Models are compared to each other
* The comparison contains at least a statement about a LLMs
  * size
  * adaptability
  * how complex the prompts to train with the LLM are (how complex will it be for the devs to work with the base model?)
  * how a new dataset must look like, so that the base model can be retrained with the new dataset (which requirements given our use case has the base model)
  * (other key decision criterias that are identified by the assignee)
* Googles Gemma Model is part of the comparison
*  A candidate selection is created
  * The selected LLMs should fulfill the following requirements
    * The LLM is available under a permissive license
    * English must be supported
  * The selection should include at least 3 base models
  * Recommendations are made with which base model we should start training (like f.e. a ranking of the base models with an explanation for the ranking)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dominic0df** [comment]
My PO collegue sliced this issue into 3 other parts, added some other requirements and put them into the backlog, so this issue is no longer required.
